Python/sysmodule.c

```diff
@@ -3358,6 +3358,16 @@ make_impl_info(PyObject *version_info)
     if (res < 0)
         goto error;
 
+#ifdef VPLATFORM
+    value = PyUnicode_FromString(VPLATFORM);
+    if (value == NULL)
```

Comment:
Please add braces (PEP 7), here and below.

Reply:
But it seems the guidelines mention that we should be consistent with the existing code style rather than blindly following the rules, if I remember correctly.

If this needs to be added, perhaps I need to modify all instances within the function.